packages/node/.env.defaults

```diff
@@ -4,3 +4,9 @@ FIREBASE_MESSAGING_SERVER_KEY="messageKey"
 FIREBASE_STORE_SERVER_KEY="storeKey"
 FIREBASE_STORE_PREFIX_KEY="dev-store"
 GANACHE_PORT="8545"
+POSTGRES_USER="postgres"
```

Comment:
It feels a bit odd that we have default environment variables for the `node` package that have to do with plugins being used by it (presumably in tests?)